astropy/io/misc/tests/test_parquet.py

```diff
@@ -546,7 +546,7 @@ def test_fail_meta_serialize(tmp_path):
 
 def assert_objects_equal(obj1, obj2, attrs, compare_class=True):
     """Convenient routine to check objects and attributes match."""
-
+    __tracebackhide__ = True
```

Comment:
Another one, both for not hidiing and for refactoring...

Reply:
I'll keep the refactoring for a follow up if that's alright, but I'll revert this change